### PR TITLE
Significantly improved the quality of Simplified Chinese translation.

### PR DESCRIPTION
**Pull request by f0rb1d**

### CODE REVIEW

File: core/src/main/resources/cn/message.yml [hunk 1]
```diff
@@ -1,318 +1,322 @@
...
-      &7显示 &a%s0&7 原理图来自 &a%s1&7:
-      &8 - &a左键单击 &7用于设置剪贴板的结构
-      &8 - &a右键点击&7将结构添加到多剪贴板
-      &8 - &7使用 &a%s2&7 回到世界
-    schematic_move_exists: '&c%s0 已经存在'
+      &7正在显示来自 &a%s1&7 的 &a%s0&7 个原理图:
+      &8 - &a左击&7将结构复制到您的剪贴板
+      &8 - &a右击&7将结构添加到多重剪贴板
+      &8 - &7使用 &a%s2&7 回到原世界
+    schematic_format: 可用格式 (名称: 搜索名称)
```
Comment (N0tMyFaultOG) [inline core/src/main/resources/cn/message.yml]:
```suggestion
    schematic_format: '可用格式 (名称: 搜索名称)'
```

File: core/src/main/resources/cn/message.yml [hunk 2]
```diff
@@ -1,318 +1,322 @@
...
-  sel_sphere: 球体选择器：左击=中心，右键单击设置半径
-  sel_cylindrical: 圆柱选择器：左击=中心，右键单击扩展.
-  sel_max: '%s0 最高点数.'
-  sel_fuzzy: 模糊选择器:左键单击以选择所有或有块，右键单击添加.要选择一个空气腔，请使用 //pos1.
-  sel_convex_polyhedral: 凸多边形选择器：左键=第一个顶点，右键单击以添加更多.
-  sel_list: 有关选择类型的列表使用:&c //sel list
-  sel_modes: '选择以下模式之一:'
+  sel_cuboid: '立方体: 左击点 1, 右击点 2'
+  sel_cuboid_extend: '立方体: 左击起始点, 右击扩大'
+  sel_2d_polygon: 2D 多边形选择器: 左/右点击添加点。
```
Comment (N0tMyFaultOG) [inline core/src/main/resources/cn/message.yml]:
```suggestion
  sel_2d_polygon: '2D 多边形选择器: 左/右点击添加点。'
```

File: core/src/main/resources/cn/message.yml [hunk 3]
```diff
@@ -1,318 +1,322 @@
...
+  sel_ellipsiod: '椭圆体选择器: 左击选定中心，右击扩大'
+  sel_sphere: '球体选择器: 左击选定中心，右击设置半径'
+  sel_cylindrical: '圆柱选择器: 左击选定中心，右击扩大'.
```
Comment (N0tMyFaultOG) [inline core/src/main/resources/cn/message.yml]:
```suggestion
  sel_cylindrical: '圆柱选择器: 左击选定中心，右击扩大'
```

**Comment by N0tMyFaultOG** [comment]
Thanks :)

**Comment by f0rb1d** [comment]
@N0tMyFaultOG 
Sorry for inconvenience.

**Pull request merged**